README.md

```diff
@@ -158,6 +158,90 @@ This project is optimized for use with [GitHub Codespaces](https://github.com/fe
 
 > _Note: The URLs above are just examples. The URLs will be different for your fork. The ports however will be the same._
 
+## Developer Guide (Website)
+
+The project has a  [Developer Guide](./packages/docs/website/README.md) implemented as an interactive website using the [Docusaurus](https://docusaurus.io) platform. 
+
+### 1 | Preview Website 
+
+Read the [website/README](./packages/docs/website/README.md) more details on setting up and building this package. For a quickstart, we recommend the following to _get a local dev preview_ of the guide.
+
+```bash
+# Change to the packages/docs/website directory
+$ cd packages/docs/website
+
+# Install dependencies (verify you have Node.js v18+)
+$ node --version
+v18.17.0
```

Comment:
Doesn't this version conflict with the one required to mitigate issues with Azure core tools? 18.15.0?

Reply:
The global config is under https://github.com/Azure-Samples/contoso-real-estate/blob/main/.nvmrc

Reply:
Updated the README to simplify this (removed this line, kept commands succinct)
Also updated test screenshot image (to reflect cached interactive version for consistency)

Saw package already has a `start:website` script (focus on SWA) so didn't one one for this package. Would something like this be appropriate? I can update if so (verified it works)
    "start:docs": "cd packages/docs/website && npm run start",

If that is not required, then hopefully this can be merged. cc @anfibiacreativa